script/foreman-debug

```diff
@@ -39,20 +39,20 @@ Please note that the rsync transmission is UNENCRYPTED.
 UPLOADUSAGE
 }
 
-# filter for patterns like password=XYZ or secret: abc!@#$123
+# Filter for patterns like password=XYZ, -storepass XYZ or secret: XYZ
 FILTER_WORDS=(
 password
 PASSWORD
 default_password
+key
```

Comment:
This causes paths in installer answer files or Foreman's own settings.yaml (e.g. ssl_priv_key) to be filtered out, e.g. `client_ssl_key`, is that intentional?